en/docs/get-started/quick-start-guide.md

```diff
@@ -76,7 +76,7 @@ Follow this procedure to further develop the API by adding a new resource and si
      |-----------------|-----------------------|
      | **HTTP Method** | `GET` |
      | **Path** | `personalizedMessage` |
-     | **Return Type** | `string` |
+     | **Return Type** | `string|error` |
```

Comment:
The return type value is not getting reflected properly when viewing the file content.
<img width="1406" alt="Screenshot 2022-07-11 at 11 53 27" src="https://user-images.githubusercontent.com/9528121/178201549-873c3ebc-8ec8-4210-916d-5703a5651fc6.png">

Reply:
First, I tried to replace this table with an html table to fix this, but it was not getting rendered, and the html code was showing instead of the table.

Then I tried to present this as a list of key-value pairs,. and did the same to the other tables to be consistent, but it was not looking good.

Felt that replacing each table row with sub-steps would make this guide too wordy.

Therefore, replaced only the affected table row with a sub step. This too seems a bit inconsistent.

Please share any other approaches you can think of to address this in a better way

Reply:
Fixed by adding a \ before the pipeline as suggested by Nisrin.